wit/keyvalue.wit

```diff
@@ -0,0 +1,26 @@
+package wasmcloud:keyvalue@0.1.0
+
+/// An interface for reading and writing keys and their respective values
+interface readwrite {
+    get: func(key: string) -> result<option<string>, error>
```

Comment:
I know the goal of phase 1 was to match the current contracts and this interface doesn't support all the set and list stuff we currently have. I am (obviously) perfectly ok with doing it this way, but just wanted to make sure that won't violate the spirit of what you're trying to do with phase 1